web/src/modules/public/markets/sagas/tickersSaga.ts

```diff
@@ -0,0 +1,33 @@
+import { call, put } from 'redux-saga/effects';
+import { sendError } from '../../../';
+import { API, RequestOptions } from '../../../../api';
+import { marketsTickersData, marketsTickersError } from '../actions';
+
+const tickersOptions: RequestOptions = {
+    apiVersion: 'peatio',
+};
+
+export function* tickersSaga() {
+    try {
+        const tickers = yield call(API.get(tickersOptions), `/public/markets/tickers`);
+
+        if (tickers) {
+            const pairs = Object.keys(tickers);
+
+            const convertedTickers = pairs.reduce((result, pair) => {
+                result[pair] = tickers[pair].ticker;
```

Comment:
What if there will be no nested `ticker` in the structure?
1. Define types or use existing interface.
2. Separate to another function and cover with unit tests.